src/components/Settings/SettingsMain.tsx

```diff
@@ -24,29 +27,27 @@ const messages = defineMessages({
   applicationTitle: 'Application Title',
   applicationurl: 'Application URL',
   region: 'Discover Region',
-  regionTip:
-    'Filter content by region (only applies to the "Popular" and "Upcoming" categories)',
```

Comment:
Are these tips no longer relevant?

Reply:
I didn't remove the tip, just the parts in parentheses.  We are also filtering results for genres now, and possibly more areas in the future.  We can't possibly list them all, and I think we only filter where it makes sense, so I didn't feel the category clarification was necessary anymore.

I did change the region tip to say "filter content by **regional availability**" though, since that is a bit more clear on what the region setting actually does.